src/toil/test/provisioners/aws/awsProvisionerTest.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (C) 2015 UCSC Computational Genomics Lab
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import pipes
+
+from uuid import uuid4
+from toil.test import needs_aws, integrative, ToilTest
+
+
+log = logging.getLogger(__name__)
+
+
+@needs_aws
+@integrative
+class AWSProvisionerTest(ToilTest):
+
+    def __init__(self, methodName='AWSprovisioner'):
+        super(AWSProvisionerTest, self).__init__(methodName=methodName)
+        self.instanceType = 'm3.large'
+        self.keyName = 'jenkins@jenkins-master'
+        self.clusterName = 'aws-provisioner-test-' + str(uuid4())
+        self.toilScripts = '2.1.0a1.dev654'#'2.1.0a1.dev455'
+        self.numWorkers = 10
+        self.numSamples = 10
+        self.spotBid = '0.15'
+
+    def setUp(self):
+        super(AWSProvisionerTest, self).setUp()
+        self.jobStore = 'aws:%s:toil-it-%s' % (self.awsRegion(), uuid4())
+
+    def tearDown(self):
+        from toil.provisioners.aws.awsProvisioner import AWSProvisioner
+        AWSProvisioner.destroyCluster(self.clusterName)
+
+    def _test(self, spotInstances=False):
+        from toil.provisioners.aws.awsProvisioner import AWSProvisioner
+        AWSProvisioner.launchCluster(instanceType=self.instanceType, keyName=self.keyName,
+                                     clusterName=self.clusterName)
+
+        master = AWSProvisioner._getMaster(clusterName=self.clusterName, wait=True)
```

Comment:
Ahhh, I was confusing this call site with another. You're right, this is leftover from before launchCluster waited. I'll remove this since it's now redundant.